library/alloc/src/lib.rs

```diff
@@ -103,13 +100,10 @@
 #![feature(inplace_iteration)]
 #![feature(lang_items)]
 #![feature(layout_for_ptr)]
-#![feature(libc)]
-#![feature(map_first_last)]
-#![feature(map_into_keys_values)]
 #![feature(maybe_uninit_ref)]
 #![feature(negative_impls)]
 #![feature(never_type)]
-#![feature(new_uninit)]
+#![cfg_attr(test, feature(new_uninit))]
```

Comment:
I personally feel like putting a feature behind a `cfg` is slightly weird, but apart from that gj here :+1:

Reply:
It's done so higher up in the file as well. Should I move it up?

Reply:
wouldn't hurt I think, but I also don't think it's necessary :shrug: